package.json

```diff
@@ -6,15 +6,14 @@
   "dependencies": {
     "arp-listener": "0.0.0",
     "dominos": "^1.0.5",
-    "dotenv": "^1.2.0",
     "i": "^0.3.3",
     "node-dash-button": "^0.1.1",
     "prompt": "^0.2.14"
   },
   "devDependencies": {},
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "sudo node app.js"
+    "start": "sudo nodejs app.js"
```

Comment:
I believe there's a naming conflict with `node` on Ubuntu and other machines. If users do not have [nodejs legacy](https://packages.debian.org/sid/nodejs-legacy) installed it may call the wrong command.